clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -65,4 +73,94 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:complexType name="server-side-config-spec">
+    <xs:sequence>
+      <xs:element name="default-resource" type="tc:resource-link-spec" minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation>
+            Specifies the default server-side storage resource to use for storing cache data.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="shared-pool" type="tc:shared-pool-spec" minOccurs="0" maxOccurs="unbounded">
+        <xs:annotation>
+          <xs:documentation>
+            Defines a pool of server-side storage resource to be shared amongst multiple caches.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+
+
+  <xs:complexType name="resource-link-spec">
```

Comment:
Left the definition in place, but used the new "from" global attribute, and renamed this to be default-resource-spec.